Gopkg.lock

```diff
@@ -0,0 +1,33 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  name = "github.com/BurntSushi/toml"
+  packages = ["."]
+  revision = "b26d9c308763d68093482582cea63d69be07a0f0"
+  version = "v0.3.0"
+
+[[projects]]
```

Comment:
I don't think this is actually used I think `testify` and `toml` should be the only deps.

Can you re-run `dep ensure` @evanlimanto ?

Reply:
⚡️ removed them manually, since running `dep ensure` did nothing

Reply:
🤔 wonder why.

Reply:
discussed offline -- we think they are transitive dependencies of `testify`